plugins/@grouparoo/mailchimp/src/migrations/000001-plugin-mailchimp-syncModes.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+  up: async function (migration) {
+    await migration.sequelize.transaction(async () => {
+      // Set all existing instances to "sync" to maintain same behavior
+      await migration.sequelize.query(
+        `UPDATE "destinations" SET "syncMode"='sync' WHERE "state"='ready' AND ("type"='mailchimp-export' OR "type"='mailchimp-export-id') AND "locked" IS NULL`
+      );
+    });
+  },
+
+  down: async function (migration) {
+    // nothing to do
```

Comment:
I'm going to change this on the others that say `// nothing to do`, we should actually set the syncMode column to `null` on these destinations.